cloudformation/ad/ad-integration.yaml

```diff
@@ -708,9 +708,6 @@ Resources:
                   - iam:UploadServerCertificate
                   - iam:TagServerCertificate
                 Resource: !Sub arn:${AWS::Partition}:iam::${AWS::AccountId}:server-certificate/*
-                Condition:
```

Comment:
Why are we removing this condition?

Reply:
In commit descriptions: iso regions don't support tagging certificates. Therefore, the condition in IAM policy is too restrictive in iso reginos

Reply:
Then the restriction should be removed only if in ISO regions. Let's do this in a follow up PR